spark-rapids/spark-rapids.sh

```diff
@@ -402,6 +408,9 @@ function install_nvidia_gpu_driver() {
 
   elif is_ubuntu ; then
 
+    # Unhold NVIDIA packages to allow upgrades (see issue #1321)
+    unhold_nvidia_packages
+
     execute_with_retries "apt-get install -y -q 'linux-headers-$(uname -r)'"
```

Comment:
I still think that upgrading the kernel in an init action is folly, but the a100 still requires a reboot, so I'll allow it.  I've been talking with @jayadeep-jayaraman about a potential patch to the GPU attaching mechanism in GCE to allow a ,passthrough=true argument in addition to the type=...,count=... arguments so that we can simulate a UEFI PCIe devices reset using qemu without a reboot.

Better would be if NV could patch their driver so that nvidia-smi --reset worked for the a100s without requiring a reboot, be it simulated or real.

Reply:
I agree, thanks for checking @cjac!